package.json

```
@@ -1,13 +1,13 @@
 {
-  "name": "@apepper/scrivito-youtube-video-widget",
+  "name": "scrivito-youtube-video-widget",
   "version": "0.1.3",
```

Can you also bump the version number?
```suggestion
   "version": "0.1.4",
```